app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { getCategories } = require('./db/data/development-data/controllers/getCategories.controller');
+
+const app = express();
+
+app.use(express.json());
```

Comment:
we don't need this line yet! This line parses req.body, which isn't something you're using at this point